src/main/kotlin/com/jetbrains/snakecharm/inspections/quickfix/MoveCaretAndInsertEqualsSignQuickFix.kt

```diff
@@ -19,10 +21,13 @@ class MoveCaretAndInsertEqualsSignQuickFix(element: PsiElement) : LocalQuickFixA
             startElement: PsiElement,
             endElement: PsiElement
     ) {
+        val pythonCodeStyleSettings = CodeStyle.getCustomSettings(file, PyCodeStyleSettings::class.java)
+        val insertionString = if (pythonCodeStyleSettings.SPACE_AROUND_EQ_IN_KEYWORD_ARGUMENT) " = " else "="
+
         val offset = startElement.textOffset
         editor?.caretModel?.moveToOffset(offset)
         if (editor?.document?.isWritable == true) {
-            editor.document.insertString(offset, "=")
+            editor.document.insertString(offset, insertionString)
```

Comment:
Could we implement it here with 'red box' shown around the place where to type key? E.g. like rule rename quickfix or introduce variable action in python.

Reply:
done